content/docs/overview.md

```diff
@@ -147,51 +147,8 @@ tasks.  Tokio implements [async] syntax to improve readability of futures.
 [`hyper`]: https://hyper.rs/guides/
 [`actix`]: https://actix.rs/book/actix/
 
-# Example
-
-A basic TCP echo server with Tokio:
-
-```rust
-extern crate tokio;
-
-use tokio::prelude::*;
-use tokio::io::copy;
-use tokio::net::TcpListener;
-
-fn main() {
-# }
-# fn hax() {
-    // Bind the server's socket.
-    let addr = "127.0.0.1:12345".parse().unwrap();
-    let listener = TcpListener::bind(&addr)
-        .expect("unable to bind TCP listener");
-
-    // Pull out a stream of sockets for incoming connections
-    let server = listener.incoming()
-        .map_err(|e| eprintln!("accept failed = {:?}", e))
-        .for_each(|sock| {
-            // Split up the reading and writing parts of the
-            // socket.
-            let (reader, writer) = sock.split();
-
-            // A future that echos the data and returns how
-            // many bytes were copied...
-            let bytes_copied = copy(reader, writer);
-
-            // ... after which we'll print what happened.
-            let handle_conn = bytes_copied.map(|amt| {
-                println!("wrote {:} bytes", amt.0)
-            }).map_err(|err| {
-                eprintln!("IO error {:?}", err)
-            });
-
-            // Spawn the future as a concurrent task.
-            tokio::spawn(handle_conn)
-        });
-
-    // Start the Tokio runtime
-    tokio::run(server);
-}
-```
-
-More examples can be found [here](https://github.com/tokio-rs/tokio/tree/v0.1.x/tokio/examples).
+# Get started with Tokio
+
+If you like to read code first, complete examples can be found
+[here](https://github.com/tokio-rs/tokio/tree/*/tokio/examples), or
```

Comment:
This link 404s for me (docs.rs links above works because docs.rs has special support for it)

Reply:
Oops -- I should have tested each link.  I'll fix.  Thank you!

Reply:
fixed!